docs/book/v3/features/helpers/url-helper.md

```diff
@@ -70,6 +70,15 @@ Each method will raise an exception if:
 > zendframework/zend-expressive-helpers. Prior to that version, the helper only
 > accepted the route name and route parameters.
 
+### Other methods available
+
+- `getRouteResult() : ?Zend\Expressive\Router\RouteResult` (since
```

Comment:
I like code examples and all others too. 😉 
A simple example (one-liner) is missing here.